src/components/calcite-radio-button/calcite-radio-button.scss

```diff
@@ -2,19 +2,21 @@
   @include calcite-theme-dark();
 }
 :host {
-  display: grid;
-  grid-gap: 8px;
-  align-items: center;
   cursor: pointer;
   .radio {
     border-radius: 100%;
     box-shadow: inset 0 0 0 1px var(--calcite-ui-border-1);
     cursor: pointer;
     transition: all 0.15s ease-in-out;
   }
+}
+:host(.hasLabel) {
+  display: grid;
+  grid-gap: 8px;
```

Comment:
What is the browser support for this?

Reply:
https://caniuse.com/#feat=css-grid. Edge 16 and up, IE support is shaky, but its decent with the other vendors.